grammars/SeedAST.asdl

```diff
@@ -0,0 +1,14 @@
+-- ASDL's 4 builtin types are:
```

Comment:
Copyright info and Apache license head, please.

Reply:
Done